# KlebNetTrainingSep2025
Slides from the "Klebsiella pneumoniae Genomic Epidemiology and Antimicrobial Resistance" delivered by members of the [KlebNET Genomic Surveillance Platform](https://klebnet.org) in September 2025.

A version of these slides were presented live by KlebNET team members as part of the [GenEpi-BioTrain - Genetic Epidemiology and Bioinformatics Training Programme](https://learning.ecdc.europa.eu/enrol/index.php?id=787) run by the European CDC (ECDC). To see/hear the video recordings of the KlebNET lectures you need to enrol (free!) in the GenEpi-BioTrain and navigate to the [Klebsiella session](https://learning.ecdc.europa.eu/course/view.php?id=1151).

<img src="https://mirrors.creativecommons.org/presskit/buttons/88x31/png/by.png" alt="CC-BY" width="50"> These slides are made available under a [CC-BY 4.0 license](https://creativecommons.org/licenses/by/4.0/), which means you can distribute, remix, adapt, and build upon the material in any medium or format, so long as attribution is given to the creator. If you use anything here, please attribute the individual creator by name (listed below and in each slide deck) and use the DOI below to cite their material.

[![DOI](https://zenodo.org/badge/1073545002.svg)](https://doi.org/10.5281/zenodo.17312177)

### Lectures and Credits

**1. Introduction to _Klebsiella pneumoniae_** - [Kat Holt](https://holtlab.net/), London School of Hygiene and Tropical Medicine, UK
* Introduction to Klebsiella pneumoniae
* Overview of genomic features and strain diversity (population structure and lineages, capsule and O antigen loci, antimicrobial resistance mechanisms, virulence and hypervirulence factors)
* Challenges interpreting virulence markers and hypervirulence
* KlebNET Genomic Surveillance Platform: resources and collaborative opportunities

**2. Bacterial strain taxonomy using LIN codes** - [Sylvain Brisse](https://research.pasteur.fr/fr/member/sylvain-brisse/), Institut Pasteur, Paris, France
* Comparative genomics approaches, including cgMLST
* What are LIN codes and how they work
* Links between LIN codes and the MLST nomenclature
* Integration into public strain taxonomy databases
* Demonstration and practical exercises with BIGSdb-Pasteur

**3. In silico serotyping of _Klebsiella_** - [Tom Stanton](https://www.linkedin.com/in/tom-stanton-676556100?originalSubdomain=uk), Monash University, Australia
* What are the capsule (K) and outer LPS (O) polysaccharides?
* K and O loci
* K and O typing with Kaptive
* How to use Kaptive
* How to interpret Kaptive results

**4. _K. pneumoniae_ genome analysis with Kleborate** - [Margaret Lam](https://research.monash.edu/en/persons/margaret-lam), Monash University, Australia
* Overview of key virulence loci in _K. pneumoniae_
* Genotyping with Kleborate
* Features of Kleborate
* How to interpret Kleborate output
* Ongoing/future developments

**5. Convergence of hypervirulence and AMR** - [Margaret Lam](https://research.monash.edu/en/persons/margaret-lam), Monash University, Australia
* Overview of typical AMR and (hyper)virulence gene distribution in different _K. pneumoniae_ clones
* Trends of ESBL and carbapenemase distributions in different geographies and _K. pneumoniae_ clones
* Overview of AMR and virulence convergence
* Application of Kleborate to detect potential convergent genomes
* Global snapshot of AMR and virulence convergence from public genome data
* Challenges around the lack of consensus definition of hypervirulence

**6. Seroepidemiology of _Klebsiella_** - [Kelly Wyres](https://wyreslab.com/), Monash University, Australia
* _K. pneumoniae_ Vaccine Value Profile
* Recap on _K. pneumoniae_ K and O antigen genetics
* K and O diversity in hypervirulent and multidrug resistant clones
* K and O diversity in clinical _K. pneumoniae_ collections (geographic differences, temporal fluctuations)
* Meta-analysis of neonatal sepsis isolates
* Sero-epi web app

**7. One Health genomic surveillance of _K. pneumoniae_** - [Marit Hetland](https://www.linkedin.com/in/marit-hetland?originalSubdomain=no), Stavanger University Hospital, Norway
* One Health & why it matters for public health
* Non-human reservoirs of _K. pneumoniae_
* Studies of _K. pneumoniae_ in a One Health context
* Genomic approaches to _K. pneumoniae_ One Health surveillance
